lib/api/v3/work_packages/work_package_representer.rb

```diff
@@ -642,14 +713,17 @@ def json_cache_key
            I18n.locale,
            json_key_representer_parts,
            represented.cache_checksum,
+           timestamps,
```

Comment:
I have added specs for caching. ➝ 

Actually, adding the `timestamps` to the `json_cache_key` is not necessary currently, because the relevant json properties are already marked with `uncacheable: true`.

- I'm removing the `timestamps` from the `json_cache_key` for now.
- I've added the task of re-enabling caching in the scenarios we've [discussed today](https://matrix.to/#/!JNelJmTpjLHzTwgpaV:fiedl.net/$frC4BP9tLst2M-Fhk6qeyb10NZcfYKuEMH16o_k1G0E?via=fiedl.net&via=openproject.org) to the [follow-up steps](https://github.com/fiedl/openproject/issues/4)